llvm/include/llvm/CodeGen/TargetInstrInfo.h

```diff
@@ -913,6 +913,10 @@ class TargetInstrInfo : public MCInstrInfo {
     return false;
   }
 
+  /// Return true if the target will always try to convert predictable branches
+  /// to selects.
+  virtual bool shouldConvertPredictableBranches() const { return true; }
+
```

Comment:
The default should be `false`

Reply:
I think the current behavior is we will convert predictable branches. To not touch too much targets, I leave it to be true here.